howto/instance/create-instance.md

```diff
@@ -4,13 +4,13 @@ To launch an application or an image, Anbox Cloud creates an instance for it. To
 
 ## Using the dashboard
 
-Select *Create Instance* on the *Instances* view to launch an instance. The information required for launching an instance depends on whether you are creating the instance from an application or an image.
+*Create Instance* in the instances view to launch an instance. The information required for launching an instance depends on whether you are creating the instance from an application or an image.
```

Comment:
Should this be `Click *Create Instance*` ?

Reply:
I was trying to combine the button name with the action performed but I guess it does not read well. I have reworded to focus on the action alone.